.github/workflows/eden.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           if [ -f ${{ github.workspace }}/eve/tests/eden/eden-version ]; then
             EDEN_VERSION=$(cat ${{ github.workspace }}/eve/tests/eden/eden-version)
           else
-            EDEN_VERSION=lfedge/eden:0.8.0
```

Comment:
My understanding from @giggsoff a long time ago is that this is just a fallback and that we should update /eve/tests/eden/eden-version when we want a new version.

Reply:
I did change both `eden-version` and these fallbacks. But now I decided to revert change for fallbacks - let's do that later when we will make next eden tag.